### ISSUE TITLE
Docs say "Yarn has created a new /node_modules folder", but npm is also available in the tabs above.

### ISSUE THREAD

**Issue by sebastiancarlos** [facebook/react-native-website#4006]
## Description

In the page ["Integration with Existing Apps"](https://reactnative.dev/docs/integration-with-existing-apps), the text reads "Yarn has created a new /node_modules folder", but there is a tab just above to select either Yarn or npm.

This is found in the following source files:
- [docs/_integration-with-existing-apps-java.md](https://github.com/facebook/react-native-website/blob/2a880c5bb7ce29455b66a2f9510ad4e314430f42/docs/_integration-with-existing-apps-java.md?plain=1#L78)
- [docs/_integration-with-existing-apps-kotlin.md](https://github.com/facebook/react-native-website/blob/2a880c5bb7ce29455b66a2f9510ad4e314430f42/docs/_integration-with-existing-apps-kotlin.md?plain=1#L78)
- [docs/_integration-with-existing-apps-swift.md](https://github.com/facebook/react-native-website/blob/2a880c5bb7ce29455b66a2f9510ad4e314430f42/docs/_integration-with-existing-apps-swift.md?plain=1#L80)

## How can we address it?

Either use a generic term, remove it, mention both, or make it dynamic.

**Comment by cortinico** [comment]
Yup, we can update it to "Installation process has created..." as in the other page. @sebastiancarlos could you send a PR with the fix?

**Comment by sebastiancarlos** [comment]
> @sebastiancarlos could you send a PR with the fix?

Dear @cortinico,

Thank you for your prompt response. While I understand the collaborative nature of open-source projects and the existing expectation for contributors to submit pull requests for issues they identify, my current professional commitments fully occupy my time. Unfortunately, this means I'm not in a position to undertake additional tasks.

I recognize the importance of community involvement in enhancing and refining the project, and I brought this issue to light with the intent that it might be resolved by those with the bandwidth to do so. I'm optimistic that this matter will garner attention from other contributors who are capable of implementing the necessary changes. 

Meanwhile, I will continue to support the project in ways that align with my current resources. I'm excited to see how the project evolves and hope this documentation issue will be addressed for the betterment of all users.

Best regards,
Sebastian Carlos
